{'project_id': '3130', 'year': '1984', 'x': -1221750, 'y': 2953020, 'plot_id': '23616', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='23616', x=-1221750, y=2953020)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='23616', x=-1221750, y=2953020)
23616
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_23616_3130.p'.
Failed qa: 1984-03-23T17:46:15.7470250Z
.......Failed qa: 1984-04-18T17:58:06.3880000Z
.......Failed qa: 1984-05-04T17:57:59.145040Z
.......Failed qa: 1984-05-20T17:58:41.9800440Z
..........................................Failed qa: 1984-07-07T17:59:06.1120880Z
..............Failed qa: 1984-07-23T17:59:44.468072Z
..............Failed qa: 1984-08-03T17:41:46.8930250Z
Failed qa: 1984-08-08T18:00:17.463018Z
.......Failed qa: 1984-08-17T17:54:11.8440190Z
Failed qa: 1984-08-19T17:42:17.6980250Z
Failed qa: 1984-08-24T18:00:32.8760880Z
Failed qa: 1984-08-26T17:48:35.8950130Z
Failed qa: 1984-09-02T17:54:25.2190000Z
.....................Failed qa: 1984-10-27T18:00:32.6220440Z
.....................Data has been pickled and saved to '/efs/timesync/1984/audit/1984_23616_3130_qa_rejects.p'.
Success tracking goes here
